.eslintrc

```diff
@@ -21,6 +21,7 @@
         "location"
       ]}],
     "react/display-name" : [0],
+    "react/no-find-dom-node": [0],
```

Comment:
I remember back when `getDOMNode` was deprecated and `ReactDOM.findDOMNode` was the new thing :frowning_face: